Project.toml

```diff
@@ -55,6 +55,7 @@ DocStringExtensions = "0.8, 0.9"
 Enzyme = "0.11.14"
 FFTW = "1"
 Glob = "1.3"
+GPUArrays = "0.9.1"
```

Comment:
just fyi this does not mean "fixed to 0.9.1" but it means any version [0.9.1, 0.10.0)

Reply:
Still learning Julia (only a year under my belt on a couple projects); thanks